core/arch/x86/decode_table.c

```diff
@@ -4522,6 +4738,48 @@ const instr_info_t vex_extensions[][2] = {
   }, { /* vex ext 78 */
     {INVALID,   0x66380d18, "(bad)", xx, xx, xx, xx, xx, no, x, NA},
     {OP_vpermilpd, 0x66380d18, "vpermilpd", Vx, xx, Hx,Wx, xx, mrm|vex|reqp, x, END_LIST},
+  }, { /* vex ext 79 */
+    {OP_seto,    0x0f9010,             "seto", Eb, xx, xx, xx, xx, mrm, fRO, END_LIST},
+    {PREFIX_EXT, 0x0f9010, "(prefix ext 144)", xx, xx, xx, xx, xx, mrm,   x, 144},
+  }, { /* vex ext 80 */
+    {OP_setno,   0x0f9110,            "setno", Eb, xx, xx, xx, xx, mrm, fRO, END_LIST},
+    {PREFIX_EXT, 0x0f9110, "(prefix ext 145)", xx, xx, xx, xx, xx, mrm,   x, 145},
+  }, { /* vex ext 81 */
+    {OP_setb,    0x0f9210,             "setb", Eb, xx, xx, xx, xx, mrm, fRO, END_LIST},
```

Comment:
bug: setb is fRC!  Please double-check all the others.  Are there more pre-review tests that could be run (and prob mentioned in the commit message) to give the reviewer confidence these are correct?

Reply:
Fixed. I've checked the others manually.